src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
     public class ConsoleLogger : BaseLogger, IAnalysisLogger
     {
+        //  TODO:  We directly instantiate this logger in two classes, creating 
+        //  unamanged dependencies.  Fix this pattern with dependency injection or a factory.
```

Comment:
I don't think we have to worry in this. Since the lifetime of the use is a command, the unmanaged resource will go away when it finishes.

Reply:
As discussed, created an issue for this.

---
In reply to: [564753259](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r564753259) [](ancestors = 564753259)